src/api/contexts/contexts.go

```diff
@@ -29,6 +30,8 @@ type (
 		Chromosome string
 		Genotype   constants.GenotypeQuery
 		SampleIds  []string
+		Dataset    uuid.UUID
+		DataType   string
```

Comment:
given that gohan only provides the `variant` data type, is this necessary?

Reply:
Yeah I was debating this with myself too. I ended up keeping it since I wanted web to be able to query either katsu or gohan with the same URL path `datasets/:dataset_id/data-types/:data_type`, and to keep the door open if new data types are added to gohan. We could remove it and have the route in gohan be variant explicit `datasets/:dataset_id/data-types/variant`

Reply:
ok I'm fine with this as-is then